---
title: Biography
feature_text: |
 <span style="color:white">A little bit about me</span>
feature_image: "/pic/colorado.jpg"
excerpt: "Biography"
geometry: margin=30mm
---
<table cellpadding="10">
  <tr>
  <td colspan="3" >

  </td>
  </tr>
  <tr>
  <td width="65%">

<small>Hello, thanks for visiting my website! Here is a bit about me and my story.</small>
<br>
<br>
<h5>Beginning my research career</h5><small>
<br>
I discovered my passion for science and programming in 7th grade. My town’s local ski hill, Pajarito, sometimes didn’t get enough snow to open, and I wondered what might cause this. People claimed that El Nino was partially causing this variation in snowfall, but I wanted to put it to the test. I downloaded daily local weather data for my hometown and found that the El Nino index in the fall was a decent predictor of winter weather. 
<br>
<br>
The next year I decided to use my new programming skills to teach people about climate change, a topic that I am very passionate about. I reasoned that people feel a connection to the place where they grow up, so showing them how the climate of their hometown has changed might convince them to live in a more climate-conscious manner. I downloaded daily weather data from every weather station in the world since 1960 and calculated how each town’s climate has changed in both the mean and in extremes (e.g. number of heat waves, number of floods). Through this project I learned that my work can have real impacts on the lives of people around me and the decisions that they make.
<br>
<br></small> 
<h5>High school overview</h5><small>
<br>
After personally studying the rapid rate of climate change, I became focused on how it will impact people’s lives, and realized that its most important impact may be on food production. I determined historical relationships between heat extremes and crop yields for every county in the US, then used these models to predict US crop production to 2100, taking into account different emissions scenarios and the rate of technological increase. Encouraged by scientists who listened to my talks, I wrote up my first scientific paper at the age of 14 and, after a few hurdles, eventually published my manuscript in the peer-reviewed journal Climate.
<br>
<br>
I wanted to expand my research to reach under-resourced communities that don't have access to reliable monitoring and reporting of weather and crop health. I turned to satellite imagery, which is available everywhere, and developed an early warning system that can predict crop yields 3-4 months before the harvest anywhere in the world. I first conducted a thorough validation in Illinois, where there is reliable reporting of crop yields, and showed that my system could predict crop yields with a median error of approximately 6%. I then applied my tool to every country in Africa. Crop prediction in Africa is significantly harder than in the US due to lack of ground truth data, variable growing seasons and sometimes multiple growing seasons in one year, small crop fields (often smaller than one pixel) that often grow more than one crop, and significant cloud cover in central African countries. I created a simple, widely-applicable tool that was able to overcome all of these challenges. When I predicted future crop production for the 2018 season then later compared my predictions to reported values, 40% of the predictions had less than a 5% error.
<br>
<br>
For this work, I was invited to give seminars around the world, including at USAID, the USDA, the International Food Policy Research Institute (IFPRI), the Group on Earth Observations Global Agricultural Monitoring Initiative (GEOGLAM), and the Famine Early Warning Systems Network (FEWS Net). I was also an invited oral presenter at the Consortium of International Agricultural Research (CGAIR) Annual Meeting in both 2019 and 2020, and served as a panelist on their Data in a Crisis Climate special. I published this work in the peer-reviewed journal Remote Sensing, and for it I won many awards, including first palce at the Regeneron Science Talent Search in 2020.
<br>
<br>
In my junior year of high school I took biology for the first time and witnessed a new world open up in front of me. I was fascinated as I learned about the mysteries of “junk DNA”, how proteins are marked for transport in the Golgi Apparatus, and how fast and precisely a cell’s processes can react to an infection. The world of small molecules enthralled me, and I came to realize that I wanted to devote my life to understanding it.
<br></small> 
<h5>College</h5><small>
<br>
I entered Harvard in 2020 and focused on deepening my knowledge of math, computation, and biology. Through my classes I advanced my mathematical skills (e.g. statistics & inference, linear algebra, and differential equations), learned fundamental programming skills such as systems programming and algorithms, and discovered the exciting world of machine learning. I chased my passion for molecular biology in classes such as immunology, epigenetics, and molecular medicine. After COVID restrictions relented, I made sure to take lab-based classes to learn how to conduct wet-lab experiments.
<br>
<br>
In the winter of my freshman year I took a class called “Analysis of Genetic Association” taught by Professor Liang. I was excited by how genome-wide association studies (GWAS) can help us understand and diagnose diseases. Around the same time, I learned about Polycystic Ovarian Syndrome --- a complex disease that is still not very well understood. Although PCOS is highly heritable, its genetic causes have remained elusive. So a classmate and I proposed a project to Professor Liang to study this complex disease using statistical genetics.
<br>
<br>
First we created the largest to-date meta analysis of PCOS by combining previous GWASs, thereby increasing the power. Through this we discovered four new genetic loci that significantly increase risk for developing PCOS. We then focused on the interactions between PCOS and inflammation, and how these relationships are mediated by obesity and sex hormones. We created the largest to-date meta analyses of the genetic signal of 138 inflammatory biomarkers, a resource that will be valuable for the future study of a wide range of complex diseases. Linkage dissociation analysis showed us that PCOS, obesity, and SHBG have similar inflammatory profiles, and causation analysis (Mendelian randomization) suggested that SHBG and obesity are causal for PCOS, and that SHBG is in turn regulated by dozens of inflammatory biomarkers. Finally, we created a model that combined these genetic relationships to better predict risk for developing PCOS based on genetics.
<br>
<br>
Although important, the results of the PCOS paper often frustrated me because they didn’t reveal many insights about mechanisms. For example, the PCOS GWAS revealed four new genetic loci that are likely causal for PCOS. However, this doesn’t reveal why they are causal, or even which SNPs are causal. Hundreds or thousands of SNPs may lie within a single LD locus that is associated with a trait, and out of those probably only one or a few are causal. Most genetic variants lie outside of protein-coding genes, but the rules of how genetic alterations mechanistically causes changes in phenotype is still very poorly understood. I believe that understanding the mechanisms of gene regulation are of utmost importance, because it lies at the base of cellular biology.
<br>
<br>
So I decided to dive into the mechanisms of gene regulation and started working with Professor Jason Buenrostro. Gene transcription is largely regulated by a network of ~1600 transcription factors, proteins that bind to specific sites in the genome then activate or repress nearby genes by recruiting cofactors. TFs drive cell differentiation and immune and stress responses, but very little is understood about how TFs work.
<br>
<br>
Protein language models (PLMs) have recently made large advancements in understanding the structure and function of various proteins. These models work because amino acids must follow a grammar in order to create a viable protein, just like words must follow grammar to create a viable sentence. It is still a question, though, whether PLMs can understand TFs, since TFs are unique due to their large intrinsically disordered regions.
<br>
<br>
I am currently fine-tuning protein language models, such as ESM2, on transcription factors to understand how well current protein language models understand transcription factors and their disordered regions, and to apply them to help engineer better transcription factors.
<br>
<br></small> 
<h5>Hobbies and Interests</h5><small>
<br>
I am an avid long-distance runner and am currently president of the Harvard Running Club. In the spring of 2023, I organized an official indoor track meet for 500 athletes from 25 running clubs in the Northeast.
<br>
<br>
I love traveling and recently discovered a love of history. In my freetime I enjoy listening to history books on audible and learning about the long and complex history of the world's civilizations. I also love exploring new places, hiking, and camping.
<br>
<br>
I always love learning new skills, and have even dabbled in some instrument engineering, as shown in the picture of the Goat Guitar!
</small> 

  </td>
  <td width="2%">
  </td>
  <td width="33%" ><img src="/pic/puppy_photo.jpg" >
  <img src="/pic/Goat_guitar.jpg" >
  <img src="/pic/Grand_canyon_run.jpg" >
  <img src="/pic/skiing.jpg" >
  <img src="/pic/post_half_marathon.jpg" >
  <img src="/pic/fountains_pic.jpg" >
  <img src="/pic/chicken.jpg" >
  </td>
  </tr>
</table>

<p align="right">
<a href="http://www.hitwebcounter.com">
<img src="http://hitwebcounter.com/counter/counter.php?page=6931336&style=0006&nbdigits=5&type=ip&initCount=0" title="unique hits since April 2018" border="0" ></a>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-117520873-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-117520873-2');
</script>

